projects/crypto-currency/iluminados/api/src/routes/index.js

```diff
@@ -3,9 +3,9 @@ import cors from 'cors'
 import dotenv from 'dotenv'
 import market from './modules/market.js'
 import feeds from './modules/feeds.js'
-import coinOfTheDay from './modules/coin-of-the-day.js'
+import CoinOfTheDay from './modules/coinoftheday.js'
```

Comment:
Seguindo o padrão dos outros imports, não deveria começar com minúsculo? `CoinOfTheDay` -> `coinOfTheDay`
```suggestion
import coinOfTheDay from './modules/coinoftheday.js'
```